DiscordBot/Commands/TicTacToeCommand.cs

```diff
@@ -0,0 +1,191 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using DSharpPlus.Entities;
+using Microsoft.CSharp.RuntimeBinder;
+using TicTacToe;
+using static TicTacToe.SquareState;
+using static TicTacToe.GameState;
+
+namespace DiscordBot.Commands
+{
+    public class TicTacToeCommand : ICommand
+    {
+        private TicTacToeManager Manager { get; set; }
+
+        private Dictionary<char, SquareState> translationDict = new Dictionary<char, SquareState>
+        {
+            {'❌', X},
+            {'⭕', O},
+            {'⬜', Empty},
+        };
+
+        private string InvalidText = "Invalid screen";
+        private string WinText = "You won!!";
+        private string LoseText = "You lost :(";
+        private string DrawText = "Game drawn";
+
+        public TicTacToeCommand(TicTacToeManager manager)
+        {
+            Manager = manager;
+        }
+
+        public async Task Run(DiscordMessage msg)
+        {
+            var splitMsg = CommandParser.Split(msg.Content);
+
+            switch (splitMsg[1])
+            {
+                case "start":
+                    await msg.RespondAsync(StartText(msg));
+                    break;
+
+                case "stop":
+                    await msg.RespondAsync(StopText(msg));
+                    break;
+
+                case "help":
+                    await msg.RespondAsync(HelpText());
+                    break;
+
+                default:
+                {
+                    var game = Manager.GetGame(msg);
+
+                    if (Manager.GameExists(game))
+                    {
+                        IEnumerable<string> textIterator = TurnText(msg, game);
+
+                        foreach (string text in textIterator)
+                        {
+                            await msg.RespondAsync(text);
+                        }
+                    }
+
+                    break;
+                }
+            }
+        }
+
+        private string HelpText() =>
+            "Each turn the bot will send a tic tac toe screen." +
+            " Place an :x: wherever you want to play and send the screen." +
+            "\nMake sure to avoid spaces and other unnecessary characters." +
+            "\nFirst to complete a row, column, or diagonal wins.";
+
+        /// <summary>
+        /// The correct text to send after the user tried to start the game
```

Comment:
Not really what the method does though is it?
It also actually starts the game, by talking to the manager.

Also, the method description can't be a noun, the method is not **The correct text**, but **Returns the correct text**